events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
+  - CFP & schedule info
+  - Social venue and registration info
+  - and more
+
+## Emeritus KCS Comms Lead Overview
+After completing a cycle as a KCS Comms Lead, the former lead takes on emeritus status. As an Emeritus KCS Lead, the former lead is expected to mentor the following KCS Lead(s). The Emeritus Lead can provide mentorship for as long as they like, but it's especially valuable for the previous lead to mentor their direct successor.
+
+### Requirements
+- Has completed at least 1 cycle as a KCS Comms Lead
+- [Kubernetes Org Membership]
+
+### Tasks
+- Provide mentorship to the KCS Comms Lead
+
+## Comms Rules and Guidelines
+
+### Social Media Rules
+- Send KCS comms to k8scontributors accounts on whatever platforms they exist on. At time of writing, that’s “X” and Mastodon. Social media should be sent only to contributor-facing accounts.
+  - Kubernetesio on X and the Kubernetes Linkedin accounts should not be used because their audiences include end users and the general public..
+- Make sure comms are formatted in Arial, 11pt, with single line spacing before copying into the target platform. This will ensure that the text looks consistent wherever it is posted.
+- If you are scheduling comms to go out at a specific time, make sure that time is aligned with the timezone the event will be held in.
+- Link to k8s.dev/events and subpages, or email messages, wherever possible. Avoid sharing links to contributor-specific links such as Registration, CFP, and GitHub Issues.
+  - While our Social Media presence is largely targeted at contributors, all social media posts are public. As such, we should avoid sharing links which could cause unwanted spam/interactions.
+- Review the Contributor Comms team’s Social Media Guidelines.
+- All social media should have at least 1 review, preferably by a shadow. A review by a social media approver is a given.
+- Generally, social media should be ready for review at least 1 day before it needs to be sent.
+- Generally, an email should be ready for review around 3 or more days before it needs to be sent.
+- If any shadow does not have Buffer access, let the KCS Comms lead know.
+- Always consider your intended and actual audience! When a message is only applicable to org members (which is pretty much - everything for the contributor summit), state that clearly!
+
+
+### Email Tips and rules
+- NOTE: A sender’s first email to a mailing list will get stuck in the moderation queue. Include around a day in the delivery estimate to account for the time a sender’s first email will spend in the moderation queue. Shadows & Volunteers should plan to send their first email to a given mailing list (leads, dev, etc) early in the day on the day it’s due. (per the timezone of the location where the summit will be held).
+- Make sure comms are formatted in Arial, 11pt, with single line spacing before copying into the target platform. This will ensure that the text looks consistent wherever it is posted.
```

Comment:
Since it's mentioned twice, what's the problem we're trying to prevent here?

Reply:
Regarding the moderation queue, we added this information to the comms document after my first email got stuck in the queue. At the time, we didn't realize this and thought the email hadn't been sent. It took some time to reach out to the leads and ask them to approve the email, so we mentioned this to account for the delay during a shadow's first email.

On the formatting specifics, we included these guidelines because, when my first email was sent out, it looked fine in the document. However, after sending, we noticed inconsistencies in styling. This happened because I drafted the email in my personal doc and copied it over to the comms doc, which was initially fine, but adding extra information directly into the draft in the comms doc led to formatting issues. These were not very noticeable in the Google Doc, and I missed them. Therefore, we thought it would be helpful to specify formatting details (Arial, 11pt, single line spacing) to ensure consistency in all communications.

However, I agree that we could replace these specifics with a more general guideline and perhaps use this example to illustrate the point.